torch/csrc/jit/passes/peephole.cpp

```diff
@@ -123,8 +122,8 @@ void PeepholeOptimize(Block * block) {
   }
 }
 
-void PeepholeOptimize(std::shared_ptr<Graph>& graph) {
-  PeepholeOptimize(graph->block());
+void PeepholeOptimize(std::shared_ptr<Graph>& graph, bool addmm_fusion_enabled) {
```

Comment:
Some comments explaining why this argument exists in the first place would be very useful.